.github/workflows/reusable-ubuntu-ci.yml

```diff
@@ -52,6 +52,11 @@ on:
         required: false
         type: boolean
         default: false
+      add-label:
+        description: 'Add the CI-PENDING label to the PR'
+        required: false
+        type: boolean
+        default: true
```

Comment:
I would default this to false, so we dont need to change the nightly / weekly jobs